Makefile

```diff
@@ -232,7 +234,7 @@ build/src/overlays/%.o: src/overlays/%.c
 
 build/src/%.o: src/%.c
 	$(CC) -c $(CFLAGS) $(MIPS_VERSION) $(OPTFLAGS) -o $@ $^
-	$(CC_CHECK) $^
+	#$(CC_CHECK) $^
```

Comment:
Is there a reason for this change?

Reply:
I think that was done by accident or as part of some test I didn't revert afterwards.

Reply:
TBH I might have been the one who did it but thanks for fixing it for me.